support-frontend/assets/components/StripePaymentRequestButton/StripePaymentRequestButtonContainer.tsx

```diff
@@ -38,32 +30,26 @@ interface PrbObjects {
 
 // ----- Component ----- //
 
-function StripePaymentRequestButtonContainer(
-	props: PropTypes,
-): JSX.Element | null {
+function StripePaymentRequestButtonContainer({
+	country,
+	contributionType,
+	isTestUser,
+	amount,
+	renderPaymentRequestButton,
```

Comment:
Thought I'd destructure the props whilst I was at it to be more inline with how we write new components